src/filters/time.js

```diff
@@ -86,14 +86,14 @@ example: |
   {{ pubdate | getDate('iso') }}
 params:
   date:
-    type: date | string
+    type: date | string | 'now'
```

Comment:
I don't see `"now"` being handled anywhere? So either remove until we want it, or handle it below?